Model/Menu/Node.php

```diff
@@ -97,14 +98,24 @@ public function setType($type)
      */
     public function getContent()
     {
-        return $this->_getData(NodeInterface::CONTENT);
+        $content =  $this->_getData(NodeInterface::CONTENT);
+
+        if (strpos($content, '{') === 0 && strpos($content, '}') === strlen($content) - 1) {
```

Comment:
`$content` is not always a string, when `_getData()` returns null this will throw an error